Documentation/observability/metrics.rst

```diff
@@ -650,6 +653,9 @@ This metric supports :ref:`Context Options<hubble_context_options>`.
 ``http``
 ~~~~~~~~
 
+Deprecated, use ``httpV2`` instead.
```

Comment:
Should there be a note in Documentation/operations/upgrade.rst about this deprecation? Is this supposed to be removed in a future version?

Reply:
Good catch, added a deprecation notice in `upgrade.rst`, removal is not planned for now.